Libs/vtkITK/vtkITKArchetypeImageSeriesReader.cxx

```diff
@@ -823,10 +830,114 @@ int vtkITKArchetypeImageSeriesReader::RequestInformation(
     this->Dictionary = itk::MetaDataDictionary();
     }
   ParseDictionary();
+
+  // Store the meta data scalar range if it exists
+  std::string range_keys[2] = {"",""};
+  this->GetScalarRangeMetaDataKeys(imageIO, range_keys);
+  std::string min_str;
+  std::string max_str;
+  bool hasMin = itk::ExposeMetaData<std::string>(this->Dictionary, range_keys[0], min_str);
+  bool hasMax = itk::ExposeMetaData<std::string>(this->Dictionary, range_keys[1], max_str);
+  if (hasMin && hasMax)
+    {
+    this->MetaDataScalarRangeMinima.clear();
+    this->MetaDataScalarRangeMaxima.clear();
+    double value = 0.0;
+    std::istringstream min_sstr(min_str);
+    while (min_sstr >> value)
+      {
+      this->MetaDataScalarRangeMinima.push_back(value);
+      }
+    std::istringstream max_sstr(max_str);
+    while (max_sstr >> value)
+      {
+      this->MetaDataScalarRangeMaxima.push_back(value);
+      }
+    }
+
   return 1;
 }
 
 //----------------------------------------------------------------------------
+void vtkITKArchetypeImageSeriesReader::GetScalarRangeMetaDataKeys(itk::ImageIOBase::Pointer imageIO,
+                                                                  std::string range_keys[2])
+{
+    if (dynamic_cast<itk::NrrdImageIO*>(imageIO.GetPointer()))
```

Comment:
Answer provided by @thewtex : there used to be an issue on Mac but that was addressed with ITK 4.11 ([here](http://review.source.kitware.com/#/c/21805/)) and Slicer is there.